pycbc/conversions.py

```diff
@@ -1009,16 +1017,23 @@ def tau_from_final_mass_spin(final_mass, final_spin, l=2, m=2, nmodes=1):
 # frequency and damping time to mass and spin.
 _berti_spin_constants = {
     (2, 2): (0.7, 1.4187, -0.4990),
+    (2, -2): (1.6700, 0.4192, 1.4700),
+    (2, 1): (-0.3, 2.3561, -0.2277),
+    (2, -1): (2.0000, 0.1078, 5.0069),
+    (2, 0): (4.0000, 1.9550, 0.1420),
     (3, 3): (0.9, 2.343, -0.4810),
+    (3, -3): (2.5500, 0.6576, 1.3378),
     (4, 4): (1.1929, 3.1191, -0.4825),
-    (2, 1): (-0.3, 2.3561, -0.2277)
+    (4, -4): (3.4000, 0.8696, 1.4074),
     }
 
 _berti_mass_constants = {
     (2, 2): (1.5251, -1.1568, 0.1292),
+    (2, -2): (0.2938, 0.0782, 1.3546),
+    (2, 1): (0.6, -0.2339, 0.4175),
+    (2, 0): (0.4437, 0.0739, 0.3350),
     (3, 3): (1.8956, -1.3043, 0.1818),
     (4, 4): (2.3, -1.5056, 0.2244),
-    (2, 1): (0.6, -0.2339, 0.4175)
     }
```

Comment:
More to do: comment out the -m and 2,0 constants for now, since we're not certain how well they work.

Reply:
Removed.